ui/component/viewers/videoViewer/internal/plugins/videojs-overlay/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright Brightcove, Inc.
```

Comment:
This one doesn't need to be included.

Reply:
Will that be the one line, or the entire file?